lib/puppet/jenkins/okjson.rb

```diff
@@ -1,4 +1,4 @@
-# encoding: UTF-8
```

Comment:
Any reason to remove this? It also looks like this code was vendored in. Perhaps we should keep it the same and ignore it in rubocop?

Reply:
Need support in modulesync config for this.  Otherwise... https://github.com/voxpupuli/puppet-jenkins/pull/877/files#diff-11a0d7906801d9dea0eccb85667ad811L17

Reply:
copied that over